src/components/SettingsAdvanced.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+    <div class="kiwi-settings-advanced">
+        <div class="kiwi-settings-advanced-notice">{{ $t('settings_advanced_header') }}</div>
+        <form class="u-form">
+            <table class="u-table kiwi-settings-advanced-table">
+                <thead>
+                    <th style="min-width: 300px;">{{ $t('settings_advanced_name') }}</th>
+                    <th style="min-width: 86px;">{{ $t('settings_advanced_status') }}</th>
+                    <th style="min-width: 50px;">{{ $t('settings_advanced_reset') }}</th>
+                    <th style="min-width: 400px;">{{ $t('settings_advanced_value') }}</th>
+                </thead>
+                <tr v-for="setting in getSettings"
+                    :key="setting.key"
+                    :style="{'font-weight': (setting.status === 'modified') ? 'bold' : 'normal' }">
+                    <td>{{ setting.key }}</td>
+                    <td>{{ $t('settings_advanced_' + setting.status) }}</td>
+                    <td style="text-align:center;">
+                        <i v-if="setting.status === 'modified'"
+                           class="fa fa-undo reset-icon"
+                           style="cursor: pointer;"
+                           @click="resetValue($event, setting.key)"/>
+                    </td>
+                    <td>
+                        <input v-if="setting.type === 'boolean'"
+                               :checked="setting.val"
+                               type="checkbox"
+                               @click="updateSetting($event, setting.key)">
+                        <input v-else-if="setting.type === 'number'"
+                               :value="setting.val"
+                               class="u-input"
+                               type="number"
+                               @keydown="blurOnEnter($event)"
+                               @change="updateSetting($event, setting.key)"
+                               @blur="updateSetting($event, setting.key)">
+                        <input v-else
+                               :value="setting.val"
+                               class="u-input"
+                               @keydown="blurOnEnter($event)"
+                               @blur="updateSetting($event, setting.key)">
+                    </td>
+                </tr>
+            </table>
+        </form>
+    </div>
+</template>
+
+<script>
+
+import state from '@/libs/state';
+import _ from 'lodash';
+
+export default {
+    data: function data() {
+        return {
+            ignoreKeys: ['emojis', 'themes', 'bnc', 'aliases', 'restricted',
+                'hide_advanced', 'windowTitle', 'startupOptions'],
+        };
+    },
+    computed: {
+        getSettings() {
```

Comment:
`getSettings` is superfluous as a computed property (as a method it would make sense). This should just be `settings`